### ISSUE TITLE
[508: COGNITION, SCREENREADER] Add older issues button disappears

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#38778]
## User Story or Problem Statement

While checking the status of a claim, a timeline of events is shown in the detail tab. If more than 5 timeline events are shown, a "See older updates" button is added below the list. When clicked, _all_ remaining events are appended to the timeline of events and the button disappears.

## Details

While testing this behavior with @CourtneyChambers77, we noted:
- The button is not associated with the timeline list (no `aria`-labels at all)
- The button should not disappear
- The number of timeline events should be limited to a small number, possibly paginated?
- The icon within the button should be hidden from screen readers
-  Focus is lost after the list is expanded
- Multiple events with the same date & description need to an ID to differentiate them from another

cc: @joshkimux

## Acceptance Criteria

- [ ] Discuss this pattern with the Design System Council and determine how the timeline events should be displayed
- [ ] Component updated to fix a11y problems and address approved pattern

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* Operating System: Windows 10
* Browser, including: Edge
* Screenreading device, if applicable: JAWS
* Server destination (staging)

## Steps to Recreate

1. Log in to ID.me with `vetsgov+test233@id.me` ([get password here](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-HLR.md))
1. Go to https://staging.va.gov/track-claims/your-claims/600219085/status in browser
1. Start screen reading device
1. Note the number of timeline items under "Claims received" and the "See older updates" button under the list
1. Activate the button and note that the button disappears and numerous new timeline additions have been appended to the timeline list
1. Note that focus is lost & that there is no way to determine how the list changed, or undo the changes

## Proposed Solution (if known)

Change the button:
- So that it's always visible
- Only adds a few timeline events at a time (e.g. 5)
- Is associated with the list via `aria-describedby`, or other needed aria-attributes (to be determined by an accessibility specialist)

Or, add pagination as needed

## Screenshots or Trace Logs

![expand](https://user-images.githubusercontent.com/136959/159085827-5c50bdfd-4138-43c3-89d1-c635c737cf68.gif)

**Comment by humancompanion-usds** [comment]
I attached a Sketch file and description of the contents to https://github.com/department-of-veterans-affairs/va.gov-team/issues/39206

**Comment by Mottie** [comment]
Claim Status Tool has been updated using the pattern @humancompanion-usds provided.

<details><summary>CST details full page screenshot</summary>

![Claim status with phase 1 - 3 expanded showing "past" updates](https://user-images.githubusercontent.com/136959/161140061-ecdaa7d0-a8f2-40e0-a0fa-2366f67112fe.png)</details>

@CourtneyChambers77 I would appreciate a final review of this in staging.

1. Log in to ID.me with `vetsgov+test233@id.me` ([get password here](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-HLR.md))
1. Go to https://staging.va.gov/track-claims/your-claims/600219085/status in browser

**Comment by jacobworrell** [comment]
assumption, this work is on staging and just needs to be QA'd

**Comment by joshkimux** [comment]
@jacobworrell correct! Will be QA'd early this week and resolved 😄

**Comment by joshkimux** [comment]
The specific issue this ticket is covering has been solved, but I'm noticing a number of other a11y issues on this page. Will close this out for the time being!
